backend/docker/fuzz/entrypoint.sh

```
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -e
+
+if [ -z "$BASE_URL" ]; then
+  echo "Error: BASE_URL environment variable is not set." >&2
+  exit 1
+fi
+
+echo "Fetching CSRF token..."
+export CSRF_TOKEN=$(curl -fsSL "$BASE_URL/csrf" | jq -r '.csrftoken')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)